Newsletters: add 344 (2025-03-07)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] News item about recent Morehouse disclosure @harding 
- [ ] ~~(Optional) Topic page about MEV @harding~~
- [x] Notable merges @Gustavojfe

LarryRuane commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+  for everyday users to use a full node to validate their own incoming
+  wallet transactions.  He asks Poinsot and other contributors to
+  consider focusing instead on making Bitcoin Core easier for everyday
+  users.  He describes the power of full node validation: those who
```
```suggestion
  users.  He describes the power of full node validation: economically-relevant users who
```
Maybe this is pedantic, but operating a full node alone doesn't help to define the consensus rules. It does little good if I run a forgotten full node in my basement that I never interact with. (It does _some_ good because it filters invalid junk from its peers so they don't have to expend bandwidth and CPU to reject the junk themselves. But this is a small effect.)

The full nodes that _economic_ users run matter. They attach _value_ to coins of one chain (fork) over another, and their full nodes enable them tell the difference.

harding replied:
I think the rest of the original sentence covered that point in a more general way: "...full nodes that validate the predominate amount of economic activity...".  The generality is required because _you_ don't need to be an economically relevant user if you operate the full node for people who are themselves economically relevant.  For example, if I operate the node that 10,000 highly active lightweight clients connect to, I don't need to perform any transactions myself in order to have significant economic relevance.

LarryRuane replied:
That's an excellent point; I hadn't considered the use of light clients.

LarryRuane commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+
+- **Private block template marketplace to prevent centralizing MEV:** Matt
+  Corallo and developer 7d5x9 [posted][c7 mev] to Delving Bitcoin about
+  allowing parties to bid in a public markets for selected space within
```
```suggestion
  allowing parties to bid in public markets for selected space within
```
Or "a public market" if singular is better.

LarryRuane commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+  miners obtain large amounts of capital and technical sophistication,
+  likely leading to them earning significantly higher profits than smaller
+  miners without those capabilities.  This will centralize mining and
+  allow the large miners to more easily censor Bitcoin transactions.
```
```suggestion
  allow large miners to censor Bitcoin transactions more easily.
```

LarryRuane commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+  - **Trusted block templates:** a miner connects to a marketplace, selects
+    the bids it wants to include in a block, and asks the marketplace to
+    construct a block template.  The marketplace responds with a block
+    header, coinbase transaction, and partial merkle branch that allow
```
```suggestion
    header, coinbase transaction, and partial merkle branch, allowing
```

harding replied:
Keeping as-is.  I think both constructions are correct and the existing construction avoids the use of a comma shortly after a comma-delimited list, making the sentence easier to skim.

LarryRuane commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+    transaction to the marketplace, which verifies the proof of work,
+    adds it to the block template, and
+    broadcasts the block.  The marketplace might include a transaction
+    paying the miner in the block template or it might pay the miner
```
```suggestion
    paying the miner in the block template, or it might pay the miner
```

harding replied:
Keeping as-is.  "It might pay" doesn't make sense here as an independent sentence, therefore it's not an independent clause, therefore this is a simple conjunction.

LarryRuane commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+    [TEE][] secure enclave, connect to marketplaces and select bids they
+    want to include in their blocks, and receive the transactions in
+    those bids encrypted to the TEE's enclave key.  The block template
+    is constructed within the TEE and it provides the host operating
```
```suggestion
    is constructed within the TEE, and provides the host operating
```

harding commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+  Anthony Towns [questions][towns pri] whether multiprocess would really
+  allow an effective split, as the individual components would remain
+  tightly coupled.  Many changes to one project would require also
+  making changes to the other projects.  However, it would be a clear
+  win to move features that don't currently require a node into a
+  library or tool that can be maintained independently.  He also
```
Leaving as-is.  The motivation for this change is unclear to me.

harding commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+  users.  He describes the power of full node validation: those who
+  operate the full nodes that validate the predominate amount economic
+  activity have the ability to define Bitcoin's consensus rules.  In an
+  example, he shows that even a 30 minute change in what rules are
```
Good reference and another interesting way to think about it.  I hadn't seen that video before.

harding commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,229 @@
+    provides it to the TEE, which verifies it and returns the full
+    decrypted block template for the miner to add to the header and broadcast.  Again, the block
+    template might include a transaction paying the miner from a UTXO
+    belonging to the marketplace or the marketplace might pay the miner
```
Same comment as above: https://github.com/bitcoinops/bitcoinops.github.io/pull/2211#discussion_r1984466124

harding commented on _posts/en/newsletters/2025-03-07-newsletter.md:
```diff
@@ -0,0 +1,269 @@
+  brings the [BOLT12][] parameter setting in line with that of [BOLT11][].
+
+- [Rust Bitcoin #4114][] lowers the minimum non-witness transaction size from 85
+  bytes to 62 bytes, aligning with Bitcoin Core’s policy (see Newsletter
```
I think this "62" is a typo that should be "65".  Will make that change.

harding commented:
Made edits for all feedback, thanks @LarryRuane !  Added lede, updated releases/RCs, added topic entries.  Skipped adding a MEV topic page but added it to https://github.com/bitcoinops/bitcoinops.github.io/issues/721#issuecomment-2705679892 .  Reviewed and made some small edits to @Gustavojfe section (thanks!).

Thanks everyone!

bitschmidty commented:
Didnt have a chance to review fully this week, just a skim, but looks great and will have to come back.

Squashed
